WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -41559,81 +41040,81 @@ b_ulpius = {
 
 b_bardewik = {
 	culture = german
-	religion = catholic
+	religion = nicene
 }
 
 b_vorrade = {
 	culture = german
-	religion = catholic
+	religion = nicene
 }
 
 b_warendorp = {
 	culture = german
-	religion = catholic
+	religion = nicene
 }
 
 b_pat_luneburg = {
 	culture = german
-	religion = catholic
+	religion = nicene
 }
 
 b_wittenborg = {
 	culture = german
-	religion = catholic
+	religion = nicene
 }
 
 # GOTLAND:
 
 b_stenkyrka = {
 	culture = swedish
-	religion = catholic
+	religion = nicene
 }
 
 b_guldsmed = {
 	culture = swedish
-	religion = catholic
+	religion = nicene
 }
 
 b_hejnum = {
 	culture = swedish
-	religion = catholic
+	religion = nicene
 }
 
 b_gildehusen = {
 	culture = swedish
-	religion = catholic
+	religion = nicene
 }
 
 b_strabain = {
 	culture = swedish
-	religion = catholic
+	religion = nicene
 }
 
 # AMALFI:
 
 b_konstantios = {
 	culture = greek
-	religion = catholic
+	religion = nicene
 }
 
 b_mauro = {
 	culture = italian
-	religion = catholic
+	religion = nicene
 }
 
 b_musco = {
 	culture = italian
-	religion = catholic
+	religion = nicene
 }
 
 b_polkarios = {
 	culture = greek
-	religion = catholic
+	religion = nicene
 }
 
 b_urso = {
 	culture = italian
-	religion = catholic
+	religion = nicene
```

Comment:
Those starts don't exist in this Mod, therefore, they should not be present in the first place.